src/js/angular/clustermanagement/templates/cluster-legend.html

```diff
@@ -0,0 +1,47 @@
+<span class="cluster-legend-wrapper">
+    <div class="cluster-legend-container">
```

Comment:
What's the difference between cluster-legend-wrapper and cluster-legend-container?! The former can be just cluster-legend and the latter cluster-nodes or whatever